pkg/app/src/components/Forms/baseForm.tsx

```diff
@@ -3,16 +3,17 @@ import { ReactNode } from 'react'
 import { Paper, Stack, Typography } from '@mui/material'
 
 interface ComponentProps {
-	title: string
+	title?: string
 	children: ReactNode
 	error?: string
+	noPadding?: boolean
 }
 
-export function BaseForm({ title, children, error }: ComponentProps) {
+export function BaseForm({ title, children, error, noPadding }: ComponentProps) {
 	return (
 		<Stack
 			component={Paper}
-			p={{ xs: 3, sm: 6 }}
+			p={!noPadding ? { xs: 3, sm: 6 } : {}}
```

Comment:
can all of this become a variant?

Reply:
I've tried it but without success but I removed the component from the proposal detail page because I don't really need it.